src/components/FooterLogo.jsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import get from 'lodash/get'
+import { withClient } from 'cozy-client'
+
+export class FooterLogo extends React.Component {
+  state = {
+    logos: []
+  }
+
+  async componentDidMount() {
+    const context = await this.props.client
+      .getStackClient()
+      .fetchJSON('GET', '/settings/context')
+    const logos = get(context, 'data.attributes.home_logos', [])
+    this.setState({ logos })
```

Comment:
Should we try / catch in order to catch network error for instance?

Reply:
I thought about it, but I don't know what we would do with the error. If the request fails we don't want to display anything, which is what happens even without the try/catch. But I could add a debug message if you think it's worth it?

Reply:
I would extract it like that:

```
const fetchHomeLogos = client => {
  try {
    const context = client
      .getStackClient()
      .fetchJSON('GET', '/settings/context')
    return get(context, 'data.attributes.home_logos', [])
  } catch (e) {
    return []
  }
}
```

This way it's clear that you are handling the error.

Nit.

Reply:
That's actually way better than my `console.warn`, thanks, i'll do that.

Reply:
You can remove the get also :)